### PR TITLE
Fixed section link to latest released version

### PR DESCRIPTION
**Pull request by svick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ You can download .NET Core SDK as either an installer (MSI, PKG) or a zip (zip,
 In order to download just the .NET Core runtime without the SDK, please visit https://github.com/dotnet/core-setup#daily-builds.
 
 > **Note:** please be aware that below installers are the **latest bits**. If you 
-> want to install the latest released versions, please check out the [section above](#download-links).)
+> want to install the latest released versions, please check out the [section above](#looking-for-v1-of-the-net-core-tooling).
```
Comment (livarcocc) [inline README.md]: It seems like you are missing a closing parenthesis here, which is breaking formatting for this file.

Comment (svick) [inline README.md]: That was intentional, there is no opening parenthesis, so there shouldn't be a closing one either.

I do see the table broken [here](https://github.com/svick/cli/blob/7ebee29dc1a231dee0e53fc44a6cc23112e945b6/README.md), but [it looks broken in the previous version too](https://github.com/svick/cli/blob/09b6be599007ea4d791dc4b7bbfa63541e819419/README.md) (which looks fine on the main page of the repo), so I don't think it's related to the parenthesis.

Comment (svick) [inline README.md]: I think the rendering issue could instead be caused by [GitHub's switch to a new Markdown parser](https://githubengineering.com/a-formal-spec-for-github-markdown/). I wasn't able to figure out how to fix it (at least not without switching from link references `[text][link-name]` to normal links `[text](http://whatever)`).

Comment (am11) [inline README.md]: Line 67 has an extra `:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|`, removing that fixes the rending problem.

Comment (svick) [inline README.md]: @am11 You're right. Thanks, fixed.

Comment (am11) [inline README.md]: @svick, no problem and thank you for sharing the link. :)
I think the new parser is hardened than the old one; as previously it was ignoring the extra column in the md table header based on it's absence in the rest of the rows.

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x86 Debug Build

**Comment by svick** [comment]
Since the table issue has been fixed by https://github.com/dotnet/cli/pull/6084, which also prevented my table fix to be merged, I have removed the table-fixing commit and rebased on top of current master.

**Comment by livarcocc** [comment]
@dotnet-bot Test Fedora24 x64 Release Build
@dotnet-bot Test OpenSUSE42.1 x64 Debug Build
@dotnet-bot Test Ubuntu16.10 x64 Debug Build

**Pull request merged**